types/plugin/range.d.ts

```diff
@@ -0,0 +1,25 @@
+import { PluginFunc, ConfigType, OpUnitType } from 'dayjs'
+import { Dayjs } from '..'
+
+declare const plugin: PluginFunc
+export = plugin
+
+declare module 'dayjs' { 
+    class DayjsRange {
+      constructor(startDate?: ConfigType, endDate?: ConfigType)
+
+      startDate: Dayjs
+      endDate: Dayjs
+
+      isRange(): boolean
+      clone(): DayjsRange
+      isOverLap(dateRange: DayjsRange): boolean
```

Comment:
```suggestion
      isOverlap(dateRange: DayjsRange): boolean
```

The capital L tripped me up for a bit trying to get the plugin working locally.